fjord/translations/gengo_utils.py

```diff
@@ -196,7 +196,17 @@ def guess_language(self, text):
                 'Accept': 'application/json'
             })
 
-        resp_json = resp.json()
+        try:
+            resp_json = resp.json()
+        except ValueError:
+            # If it's not JSON, then I don't really know what it is,
+            # so I want to see it in an error email. Chances are it's
+            # some ephemeral problem.
+            #
+            # FIXME: Figure out a better thing to do here.
+            raise GengoAPIFailure(
+                'ValueError: non-json response: {0} {1}'.format(
```

Comment:
do we need to make this a unicode string in case `resp.test` has unicode. BTW, what is `resp.test`? I was expecting `resp.body` or something.

disclaimer: I love unicode.